packages/project-disaster-trail/src/components/Game/TaskScreen/SolveScreen.js

```diff
@@ -82,6 +83,15 @@ class SolveScreen extends PureComponent {
       background: beige;
     `;
 
+    const taskImage = css`
+      background-image: url(${activeTask.imageSVG});
```

Comment:
Just a heads up, we will eventually have 2 separate images associated with a task: the task icon and the image for the task solving screen.